python/ray/data/dataset.py

```diff
@@ -527,24 +480,28 @@ def map_batches(
             ray_remote_args: Additional resource requirements to request from
                 ray for each map worker.
 
+        .. note::
+
+            The size of the batches provided to ``fn`` might be smaller than the
+            provided ``batch_size`` if ``batch_size`` doesn't evenly divide the block(s)
+            sent to a given map task. When ``batch_size`` is specified, each map task is
+            sent a single block if the block is equal to or larger than ``batch_size``,
```

Comment:
I had trouble parsing this sentence. fi the block size is larger than ``batch_size``, wouldn't the map task receive more than a single block. Wait, I guess I don't know what a block is.

Reply:
+1 I think we should just shorten this to only the first sentence.